lib/SyTest/HomeserverFactory/Synapse.pm

```diff
@@ -102,7 +102,6 @@ sub create_server
    return $self->{impl}->new( %params );
 }
 
-
```

Comment:
ftr I think that the extra line between class definitions isn't a bad thing, but :man_shrugging: